### DISCUSSION TITLE
unexpected schemeNames in xml example.

### DISCUSSION THREAD

**Discussion by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#173] [Q&A] [answered]
Hi,

in the provided xml example https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/can_24_maximal.xml, there are some schemeNames used that I do not understand:

- `<cbc:ID schemeName="InternalID">`: I do not see this schemeName defined anywhere in the sdk.
- `<cbc:ID schemeName="organization">ORG-0002</cbc:ID>`: I assume this is supposed to match the idScheme 'ORG'. How to go from 'ORG' to 'organization'?
- `<cbc:ID schemeName="organization">TPO-0001</cbc:ID>`: the schemeName references the organisations scheme, but the value is from the touchpoint scheme? 
- The last point brings up the question about how we are supposed to represent multi-level idSchemes in the xml?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Information is summarized in [identifiers section](https://docs.ted.europa.eu/eforms/1.3.1/schema/identifiers.html) of the documentation.
To answer the different questions:

- is illustrated [here](https://docs.ted.europa.eu/eforms/1.3.1/schema/procedure-lot-part-information.html#objectSection), the identifiers section should also be updated accordingly and this will be done in a next release
- illustrated in the identifiers section
- this is a mistake that should be fixed
- may be you could illustrate with a concrete example?

Thanks,
Yves

**Reply by karel-debrabandere-bosa-fgov-be** [reply]
Thank you for the clear documentation.

Is there any way to get the schemeName for a BT from the SDK instead of the docs?

**Reply by YvesJo** [reply]
Not yet
